### PR TITLE
Ben - La111.rsrc_fos_prod

### PR DESCRIPTION
**Pull request by bpbond**
#169

### CODE REVIEW

File: R/zchunk_LA111.rsrc_fos_Prod.R [hunk 1]
```diff
@@ -1,89 +1,223 @@
...
 #'
 #' @param command API command to execute
 #' @param ... other optional parameters, depending on command
 #' @return Depends on \code{command}: either a vector of required inputs,
 #' a vector of output names, or (if \code{command} is "MAKE") all
 #' the generated outputs: \code{L111.Prod_EJ_R_F_Yh}, \code{L111.RsrcCurves_EJ_R_Ffos}. The corresponding file in the
 #' original data system was \code{LA111.rsrc_fos_Prod.R} (energy level1).
-#' @details Describe in detail what this chunk does.
+#' @details For historical fossil energy production, determine regional shares of production for each primary fuel,
+#' interpolate unconventional soil production to all historical years, deduct unconventional oil from total oil,
```
Comment (mwisepnnl) [inline R/zchunk_LA111.rsrc_fos_Prod.R]: Though i like the concept of unconventional soil, change it to oil.

**Comment by bpbond** [comment]
@jhoring Do you have time & interest to review? If not I'll assign someone else, no problem.

File: R/zchunk_LA111.rsrc_fos_Prod.R [hunk 2]
```diff
@@ -1,89 +1,223 @@
...
+      select(-value.x, -value.y, -unconventionals) %>%
+      # switch the names from final to primary
+      mutate(fuel = if_else(fuel == "refined liquids", "crude oil", fuel),
+             fuel = if_else(fuel == "gas", "natural gas", fuel)) %>%
+      bind_rows(select(L111.Prod_EJ_ctry_unconvOil_Yh, GCAM_region_ID, sector, fuel, year, value)) ->
+      L111.Prod_EJ_R_F_Yh
+
+    # ------- FOSSIL RESOURCE SUPPLY CURVES
+
+    # Using supply curves from GCAM 3.0 (same as MiniCAM) (83-93)
```
Comment (jhoring) [inline R/zchunk_LA111.rsrc_fos_Prod.R]: Quick question -- what do the numbers here (83-93) and elsewhere mean?

Comment (bpbond) [inline R/zchunk_LA111.rsrc_fos_Prod.R]: See line 51 above: they're references to line numbers in the original data system file. I got the idea from @abigailsnyder .

**Comment by bpbond** [comment]
Thanks Jill!

**Pull request merged**